### ISSUE TITLE
loss_ce gradually rises after falling

### ISSUE THREAD

**Issue by CuberrChen** [hfslyc/AdvSemiSeg#66]
I was reproducing your work and a strange phenomenon emerged. The supervised cross-entropy loss(loss_ce) gradually decreases to the level of 0.4 at the beginning, but it starts to grow as the training proceeds. The evaluated Miou metrics also started to drop from the good results at the beginning. I did not modify your code much, the only difference is that I set the learning rate of S to 0.001 with 0.5 label ratio setting.

I don't particularly understand the training requirements for generative adversarial networks, but it looks as though if I set the learning rate to 0.00025, the training seems to be normal at the 0.125 label ratio setting.

For this, is there any reasonable explanation?
